### PR TITLE
Redesign of page

### PR DESCRIPTION
**Pull request by guilleSequeiraWolox**
## Summary
add Redesign of page

## Screenshot
captura de pantalla 2018-09-12 a la s 17 08 25

## Trello Card
https://trello.com/c/TBXJcGJB/12-redise%C3%B1o-de-la-p%C3%A1gina

### CODE REVIEW

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]
Try to move the border right more to the center no that close to the techo-content

File: scss/tech.scss [hunk 1]
```diff
@@ -6,40 +6,25 @@
 
 .container-technologies { 
   margin-left: 60px;
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(2, auto);
+  grid-auto-flow: row;
```
Comment (mmamaniwolox) [inline scss/tech.scss]: I think this is default prop, try without this

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

**Review by mmamaniwolox — APPROVED** [review]

File: scss/tech.scss [hunk 2]
```diff
@@ -60,3 +44,11 @@
   margin-top: 10px;
   max-height: 50px;
 }
+
+.techno-content {
+  padding-top: 150px;
+  height: 600px;
```
Comment (FrankIglesias) [inline scss/tech.scss]: Height 100%

File: scss/tech.scss [hunk 3]
```diff
@@ -60,3 +44,11 @@
   margin-top: 10px;
   max-height: 50px;
 }
+
+.techno-content {
+  padding-top: 150px;
```
Comment (FrankIglesias) [inline scss/tech.scss]: Why this padding?

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**